controllers/managedocs_controller.go

```diff
@@ -126,11 +188,51 @@ func (r *ManagedOCSReconciler) setDesiredStorageCluster(
 	if reconcileStrategy == v1.ReconcileStrategyStrict {
 		// Get an instance of the desired state
 		desired := utils.ObjectFromTemplate(templates.StorageClusterTemplate, r.Scheme).(*ocsv1.StorageCluster)
-
+		r.updateStorageClusterWithParams(desired, cs)
 		// Override storage cluster spec with desired spec from the template.
 		// We do not replace meta or status on purpose
 		sc.Spec = desired.Spec
+		r.Log.Info("StorageCluster", "desiredSpec", desired.Spec)
+		r.Log.Info("StorageCluster", "Current spec", sc.Spec)
 	}
 
 	return nil
 }
+
+func (r *ManagedOCSReconciler) updateStorageClusterWithParams(
+	sc *ocsv1.StorageCluster,
+	cs *corev1.Secret) error {
+
+	if cs == nil {
+		return nil
+	}
+	r.Log.Info("Updating storagecluster")
```

Comment:
This is a misleading log, as we are not updating the storage cluster here at all, the updating happens when we replace the sc spec